JitsiConference.js

```diff
@@ -1160,9 +1244,11 @@ JitsiConference.prototype.getPhonePin = function () {
  * for its session.
  */
 JitsiConference.prototype.getConnectionState = function () {
-    if(this.room)
-        return this.room.getConnectionState();
-    return null;
+    if(this.jingleSession) {
+        return this.jingleSession.getIceConnectionState();
+    } else {
```

Comment:
No big deal! Even with braces you don't need the else :-)  We want to eventually use the same style as we do in jitsi-meet here. We are not there yet, but ideally we should try to align all new code with the new rules. For example, this will complain for having no space after `if`.